control/plugin_manager.go

```diff
@@ -543,9 +543,12 @@ func (p *pluginManager) LoadPlugin(details *pluginDetails, emitter gomit.Emitter
 			metricTypes, err := colClient.GetMetricTypes(cfg)
 			if err != nil {
 				pmLogger.WithFields(log.Fields{
-					"_block":      "load-plugin",
-					"plugin-type": "collector",
-					"error":       err.Error(),
+					"_block":         "load-plugin",
+					"plugin-type":    resp.Type.String(),
+					"error":          err.Error(),
+					"plugin-name":    ap.Name(),
+					"plugin-version": ap.Version(),
+					"plugin-id":      ap.ID(),
```

Comment:
At this moment plugin-id is not generated yet, so api.ID() returns zero. Based on that this field might be removed from the log because does not present any valuable info.